src/services/EdFi.DataManagementService.Api.Tests.E2E/PlaywrightContext.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.Playwright;
+
+namespace EdFi.DataManagementService.Api.Tests.E2E;
+
+public class PlaywrightContext
+{
+    private static Task<IAPIRequestContext>? _requestContext;
+
+    private string _apiURL = "http://localhost:5198/";  // Backing store
+
+    public string API_URL
+    {
+        get => _apiURL;
+        set => _apiURL = value;
+    }
+
+    public IAPIRequestContext? ApiRequestContext => _requestContext?.GetAwaiter().GetResult();
+
+    public void Dispose()
+    {
+        _requestContext?.Dispose();
+    }
+
+    public async Task CreateApiContext()
+    {
+        var playwright = await Playwright.CreateAsync();
+
+        _requestContext = playwright.APIRequest.NewContextAsync(new APIRequestNewContextOptions
```

Comment:
@stephenfuqua not sure if I understand the previous question, but this is the initialization of the playwright API